SPM/.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
+/.build
+/Packages
+xcuserdata/
+DerivedData/
+.swiftpm/configuration/registries.json
+.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata
+.netrc
```

Comment:
I believe lines 6-8 are not needed as those files are stored in `~/` not in sided `SPM/`

Reply:
as I told you offline, this is a separate project (swift package) which was added to the project.
btw. it really does not matter for the sake of the Example app.

Reply:
Ok... for now :)